draft/Contest_API.md

```diff
@@ -1084,44 +1084,96 @@ Returned data:
-| email       | string         | no        | yes       | Email of team member.
+| team\_id    | ID             | no        | yes       | [Team](#teams) of this person. Required iff role is `team`.
+| name        | string         | yes       | no        | Name of the person.
+| title       | string         | no        | yes       | Title of the person, e.g. "Technical director".
+| email       | string         | no        | yes       | Email of the person.
 | sex         | string         | no        | yes       | Either `male` or `female`, or possibly `null`.
-| role        | string         | yes       | no        | One of `contestant` or `coach`.
-| photo       | array of IMAGE | no        | yes       | Registration photo of the team member.
+| role        | string         | yes       | no        | One of `contestant`, `coach`, or `staff`.
+| photo       | array of IMAGE | no        | yes       | Registration photo of the person.
 
 #### Examples
 
 Request:
 
-` GET https://example.com/api/contests/wf14/team-members`
+` GET https://example.com/api/contests/wf14/people`
 
 Returned data:
 
 ```json
 [{"id":"john-smith","team_id":"43","icpc_id":"32442","name":"John Smith","email":"john.smith@kmail.com","sex":"male","role":"contestant"},
- {"id":"osten-umlautsen","team_id":"43","icpc_id":null,"name":"Östen Ümlautsen","sex":null,"role":"coach"}
+ {"id":"osten-umlautsen","team_id":"43","icpc_id":null,"name":"Östen Ümlautsen","sex":null,"role":"coach"},
+ {"id":"bill","name":"Bill Farrell","sex":"male","title":"Executive Director","role":"staff"}
+]
+```
+
+### Accounts
+
+The accounts used for access to the contest.
+
+The following endpoints are associated with accounts:
+
+| Endpoint                       | Mime-type        | Required? | Description
+| :----------------------------- | :--------------- | :-------- | :----------
+| `/contests/<id>/accounts`      | application/json | yes       | JSON array of all accounts with elements as defined in the table below.
+| `/contests/<id>/accounts/<id>` | application/json | yes       | JSON object of a single account with elements as defined in the table below.
+
+JSON elements of problem objects:
+
+| Name              | Type    | Required? | Nullable? | Description
+| :---------------- | :------ | :-------- | :-------- | :----------
+| id                | ID      | yes       | no        | Identifier of the account.
+| username          | string  | yes       | no        | The account username.
+| password          | string  | no        | yes       | The account password.
+| type              | string  | no        | yes       | The type of account, e.g. `team`, `judge`, `admin`, `analyst`, `staff`.
+| ip                | string  | no        | yes       | IP address associated with this account, used for auto-login.
+| team\_id          | ID      | no        | yes       | The team that this account is for. Required iff type is `team`.
```

Comment:
I think the required column here should contain "depends" to reflect the "Required iff type is `team`."

Reply:
Agreed, fixed (and hopefully rebased to latest).